library/core/src/num/int_macros.rs

```diff
@@ -107,6 +107,10 @@ macro_rules! int_impl {
 
         /// Returns the number of leading zeros in the binary representation of `self`.
         ///
+        /// The
```

Comment:
request: I think it would be good to describe this more as a "see also" or "Depending what you're doing with the value, you might also be interested in …".  It needs something to emphasize that it's a *related* method, but not the same thing.  Ideally it would say why you'd want to use one or the other.